deepwell/src/services/alias/service.rs

```diff
@@ -112,6 +115,11 @@ impl AliasService {
 
                     return Err(Error::Conflict);
                 }
+
+                if slug.bytes().len() < ctx.config().minimum_name_bytes {
```

Comment:
Fyi, `string.len()` is the same as `string.bytes().len()`. However I'm fine with making it explicit since it is more intentional and compiles to the same code.